src/trove_classifiers/__init__.py

```diff
@@ -228,6 +228,7 @@
     "License :: OSI Approved :: Intel Open Source License",
     "License :: OSI Approved :: Jabber Open Source License",
     "License :: OSI Approved :: MIT License",
+    "License :: OSI Approved :: MIT No Attribution License",
```

Comment:
```suggestion
    "License :: OSI Approved :: MIT No Attribution License (MIT-0)",
```
We prefer to include the SPDX identifier for new classifiers when available.